pkg/app/piped/cloudprovider/kubernetes/kubernetes.go

```diff
@@ -41,6 +41,7 @@ const (
 	LabelResourceKey          = "pipecd.dev/resource-key"           // The resource key generated by apiVersion, namespace and name. e.g. apps/v1/Deployment/namespace/demo-app
 	LabelOriginalAPIVersion   = "pipecd.dev/original-api-version"   // The api version defined in git configuration. e.g. apps/v1
 	LabelIgnoreDriftDirection = "pipecd.dev/ignore-drift-detection" // Whether the drift detection should ignore this resource.
+	LabelSyncReplace          = "pipecd.dev/sync-option"            // Use replace instead of apply.
```

Comment:
In the future, we may add other sync options so I think it should be

```suggestion
	LabelSyncReplace          = "pipecd.dev/sync-by-replace"            // Use replace instead of apply.
```